src/modules/EventList/index.tsx

```diff
@@ -101,16 +101,15 @@ export const EventListUI = (props: EventListUIProps): ReactElement => {
                             value={filterByTypes}
                         >
                             <Tabs<EventView>
-                                className="ml-1"
+                                className="ml-1 filter_tabs"
```

Comment:
Maybe this should be `eventlist__tabs` and it can be removed from a couple lines up?

Reply:
Either that, or I think you can already do `.eventlist__tabs .tab__option` without even adding the new classname here.

Reply:
I decided to remove a couple lines up, because it is _tabs_ and not tabs + filter bar.. so changed this to `eventlist__tabs` like you said